.github/ISSUE_TEMPLATE/administration-only.yml

```diff
@@ -1,5 +1,5 @@
-name: Administration only
-description: For administrators only
+name: Direct Issue creation is not allowed
```

Comment:
Should we change the filename to `direct-issue-creation-is-not-allowed.yml` to match the new name?